### ISSUE TITLE
Resolve pose_selector warning on start-up

### ISSUE THREAD

**Issue by alexander-sung** [DFKI-NI/mobipick_labs#2]
On start-up `roslaunch tables_demo_bringup demo_sim.launch`, these warnings appear:

```
Pose_selector failed to get initial poses from configuration file. You may ignore this message if no prior poses are needed. [/pick_pose_selector_node]
Pose_selector failed to get initial poses from configuration file. You may ignore this message if no prior poses are needed. [/place_pose_selector_node]
```

Does not seem to be a problem, as the message suggests. Still isn't nice, though.

**Comment by mintar** [comment]
Let's continue the discussion in [this existing issue](https://git.ni.dfki.de/environment_representation/pose_selector/-/issues/16), since it's specific to pose_selector.

**Comment by AmosSmith3** [comment]
Addressed in DFKI-NI/pose_selector@622360874a642705722817fc655d60c1b1d83b74
